### PR TITLE
Remove Audit

### PR DESCRIPTION
**Pull request by avdev4j**
-   [x] Remove backend audit configuration
-   [x] Keep Created Date for user
-   [x] Update server cleanup.js
-   [x] Remove Angular audit components
-   [x] Remove React audit components
-   [x] Remove Vue audit components
-   [x] Update Angular cleanup.js
-   [x] Update React cleanup.js
-   [x] Update Vue cleanup.js
-   [x] Remove Audits from Language generator
-   [x] Fix Vue tests
-   [x] Fix Vue npm tests

---

Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/server/files.js [hunk 1]
```diff
@@ -357,18 +357,6 @@ const serverFiles = {
...
     serverJavaAuthConfig: [
-        {
-            condition: generator =>
-                !generator.reactive &&
-                (generator.databaseType === 'sql' || generator.databaseType === 'mongodb' || generator.databaseType === 'couchbase'),
-            path: SERVER_MAIN_SRC_DIR,
-            templates: [
-                {
-                    file: 'package/security/SpringSecurityAuditorAware.java',
-                    renameTo: generator => `${generator.javaDir}security/SpringSecurityAuditorAware.java`,
```
Comment (murdos) [inline generators/server/files.js:368]: Don't forget to remove old files in `cleanup.js`

Comment (avdev4j) [reply generators/server/files.js:368]: good catch. Thanks you

**Comment by avdev4j** [comment]
@pascalgrimaud with the merging of the vue blueprint, I have fix conflicts and rebase from master. 

I would like to apply audits removing on the vue part but I suggest to do it in another branch. This one is big enough. 

WDYT ?

**Comment by pascalgrimaud** [comment]
@avdev4j : agree. But when it's merged, Vue builds will probably fail but it's ok

**Comment by avdev4j** [comment]
Anyway, I've figured out that the build already failed because file are missing for the vue part (i18n language files), So I will manage the vue in this PR (probably this week). I add a todo checkbox ;)

**Comment by pascalgrimaud** [comment]
Ok then, it would be easy:
- npm test
- audits vue and tests
- protractor tests

**Comment by avdev4j** [comment]
Vue part done ;)

**Comment by hosseinmp76** [comment]
I can't find any issue discussing this change and why audits are removed. it would be good to mention it here as documentation.

**Comment by avdev4j** [comment]
Hi @hosseeinmp76

You can find the roadmap ticket here: https://github.com/jhipster/generator-jhipster/issues/10958

For sure this feature has been discussed between core team members, maybe on the mailing list or something like that, and we decided to add it to the roadmap.

**Comment by hosseinmp76** [comment]
@avdev4j  thank you for your reply. because v7 is published as a beta I think it would be good to know the reasons behind changes otherwise early adopters (like me) will be confused about what is happening and is it worth upgrading now or no. (in this case, I see the task " Replace old audits system by specific logs" is not done yet so maybe I should wait before a test(?) as it's not documented what is happening on audits)

**Comment by khatridreams27** [comment]
Hi all,
There is any replacement for Audits Page.
Can i add that functionality again.

**Pull request merged**